transcom/mymove
# B 20653 mobile home shipment card

## [B-20653](https://www13.v1host.com/USTRANSCOM38/story.mvc/Summary?oidToken=Story%3A982771)

## Summary

This PR updates the mobile home shipment status to 'SUBMITTED' when the shipment is submitted by the customer.
This also removes 'Please' from the shipment option description, and updates the shipment opiton description to match the PO AC.
Any labeling for Mobile home that's user facing is now "Mobile Home".

## Verification Steps for Reviewers

These are to be checked by a reviewer.


### How to test

1. Login as a customer
2. Add a mobile home shipment
3. Confirm the labeling is "Mobile Home"
4. Confirm description changes described above
5. Complete the move and sign
6. Confirm in DB that the mtoshipment status for that mobile home shipment is set to 'SUBMITTED'

### Frontend


### Backend


## ANDI Screenshots
_**Removed location Page title**_
<img width="1130" alt="Screenshot 2024-09-05 at 10 13 10 AM" src="https://github.com/user-attachments/assets/c05cf505-f6bf-4905-a7e2-67bb2cfc4114">

_**Removed Please from shipment list description**_
<img width="887" alt="Screenshot 2024-09-05 at 8 55 38 AM" src="https://github.com/user-attachments/assets/7102fd25-f6a9-483e-a525-faf258955777">

_**Change of dimension description**_
<img width="1227" alt="Screenshot 2024-09-03 at 3 36 21 PM" src="https://github.com/user-attachments/assets/5fb578c4-b69c-4558-8b75-6884413d8291">

_**Shipment Option label change**_
<img width="1340" alt="Screenshot 2024-09-03 at 3 35 26 PM" src="https://github.com/user-attachments/assets/bb9e59d1-f075-4958-bbb4-1d660e39444b">

_**Shipment Card label change**_
<img width="1723" alt="Screenshot 2024-09-03 at 3 34 36 PM" src="https://github.com/user-attachments/assets/dc506b02-6141-4277-adbf-0353fa24adb0">
